docs/resources/group_saml_link.md

```diff
@@ -0,0 +1,46 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "gitlab_group_saml_link Resource - terraform-provider-gitlab"
+subcategory: ""
+description: |-
+  The gitlab_group_saml_link resource allows to manage the lifecycle of an SAML integration with a group.
+  Upstream API: GitLab REST API docs https://docs.gitlab.com/ee/api/groups.html#saml-group-links
+---
+
+# gitlab_group_saml_link (Resource)
+
+The `gitlab_group_saml_link` resource allows to manage the lifecycle of an SAML integration with a group.
+
+**Upstream API**: [GitLab REST API docs](https://docs.gitlab.com/ee/api/groups.html#saml-group-links)
+
+## Example Usage
+
+```terraform
+resource "gitlab_group_saml_link" "test" {
+  group_id        = "12345"
+  access_level    = "Developer"
+  saml_group_name = "samlgroupname1"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `access_level` (String) Access level for members of the SAML group. Valid values are: `Guest`, `Reporter`, `Developer`, `Maintainer`, `Owner`.
+- `group` (String) The ID or path of the group to add the SAML Group Link to.
```

Comment:
You need to rerun `make reviewable` once my suggestions regarding the access levels are applied ;)